src/components/modules/dropdown/dropdown.types.ts

```diff
@@ -0,0 +1,10 @@
+import type { ButtonProps } from '@/components/elements/button/button.types';
+
+export interface DropdownProps extends Omit<ButtonProps, 'onClick' | 'href'> {
+  items: {
+    content: string;
+    as?: 'button' | 'a';
+    onClick?: () => void;
+    href?: string;
+  }[];
```

Comment:
Can we create the interface of the item and use it here:
```suggestion
  items: DropdownItems[];
```